cmd/csi-provisioner/csi-provisioner.go

```diff
@@ -326,15 +344,7 @@ func main() {
 		controller.NodesLister(nodeLister),
 	}
 
-	translator := csitrans.New()
-
-	supportsMigrationFromInTreePluginName := ""
-	if translator.IsMigratedCSIDriverByName(provisionerName) {
-		supportsMigrationFromInTreePluginName, err = translator.GetInTreeNameFromCSIName(provisionerName)
-		if err != nil {
-			klog.Fatalf("Failed to get InTree plugin name for migrated CSI plugin %s: %v", provisionerName, err)
-		}
-		klog.V(2).Infof("Supports migration from in-tree plugin: %s", supportsMigrationFromInTreePluginName)
+	if supportsMigrationFromInTreePluginName != "" {
```

Comment:
nit: maybe move this above?

Reply:
Hmm we won't be able to move it above without moving a lot of code. This is because it used provisioner options which is declared above and provisioner option uses some of the variables that are being initialized above.

Reply:
I see. It's cool to leave it here.